kolibri/plugins/coach/assets/src/modules/examCreation/handlers.js

```diff
@@ -118,6 +124,56 @@ export function showExamCreationTopicPage(store, params) {
     });
   });
 }
+export function showExamCreationBookamrksPage(store, params) {
+  return store.dispatch('loading').then(() => {
+    const { topicId } = params;
+    const topicNodePromise = ContentNodeResource.fetchModel({ id: topicId });
+    const childNodesPromise = ContentNodeResource.fetchCollection({
+      getParams: {
+        parent: topicId,
+        kind_in: [ContentNodeKinds.TOPIC, ContentNodeKinds.VIDEO, ContentNodeKinds.EXERCISE],
+      },
+    });
+    const loadRequirements = [topicNodePromise, childNodesPromise];
+
+    return Promise.all(loadRequirements).then(([topicNode, childNodes]) => {
+      return filterAndAnnotateContentList(childNodes).then(() => {
+        store.commit('SET_TOOLBAR_ROUTE', {
+          name: PageNames.EXAMS,
+        });
+        return showExamCreationPage(store, {
+          classId: params.classId,
+          bookmarksList: childNodes,
```

Comment:
Where does the call to the Bookmarks API occur in this context? This looks like it will just assign all child nodes for the topic node to the `bookmarksList` key.

Reply:
Seems like there is a missing call to the `getBookmarks` function below?